src/validation.cpp

```diff
@@ -1244,7 +1262,12 @@ MempoolAcceptResult MemPoolAccept::AcceptSingleTransaction(const CTransactionRef
                                             ws.m_base_fees, effective_feerate, single_wtxid);
     }
 
-    if (!Finalize(args, ws)) return MempoolAcceptResult::Failure(ws.m_state);
+    if (!Finalize(args, ws)) {
+        // The only possible failure reason is fee-related (mempool full). It's helpful for the
+        // caller to know what the fee and vsize were.
```

Comment:
this doesn't actually return vsize or base fees, contra to the table indications or comment here

Reply:
I got the vsize/fee field mixed up with the effective feerate field, sorry! Fixed now.

Reply:
Fixed the comment